.github/workflows/ci.yaml

```diff
@@ -1,22 +1,26 @@
-name: hcSeedBundle Test
+name: @holochain/hc-seed-bundle Test and Release
 
 on:
   push:
     branches: [ main ]
-  pull_request:
-    branches: [ main ]
```

Comment:
This appears to no longer be running unit tests on pull requests. In general we like to see passing unit tests before merging.

Reply:
fixed